docs/en/api/elements/built-in/_meta.json

```diff
@@ -33,5 +33,15 @@
     "type": "file",
     "name": "frame",
     "label": "<frame>"
+  },
+  {
+    "type": "file",
+    "name": "input",
+    "label": "<input>"
```

Comment:
Can we maybe add a `XElement` context here? So that we could directly know about it's different with other elements.